src/OptiHPLCHandler/plotting/gradient_utils.py

```diff
@@ -0,0 +1,43 @@
+def order_gradient_table(gradient_table: list[dict]) -> list[dict]:
+    """
+    Orders the gradient table by Time.
+    """
+    initial_row = None
+    other_rows = []
+    for row in gradient_table:
```

Comment:
```suggestion
other_rows = [row for row in gradient_table if row["Time"] != "Initial"]
other_rows.sort(key=lambda row: float(row["Time"])
initial_row = [row for row in gradient_table if row["Time"] == "Initial"]
result=[]
result.extend(initial_row)
result.extend(other_rows)
```

Unless you need the time as a float afterwards

Reply:
Or you could stadandardise the gradient table using the function below. Then you need to change time 0 back to Initial, though.